server/core/src/lib.rs

```diff
@@ -338,7 +339,7 @@ pub fn dbscan_restore_quarantined_core(config: &Configuration, id: u64) {
     };
 }
 
-pub fn backup_server_core(config: &Configuration, dst_path: &Path) {
+pub fn backup_server_core(config: &Configuration, dst_path: &Path, compression: BackupCompression) {
```

Comment:
Couldn't we get the compression out of the config instead of as another arg here? Or is there something I'm missing about requesting one-time compression or similar?

Reply:
That'd definitely make sense, IR dummy